probe-rs/src/probe/jlink/mod.rs

```diff
@@ -587,7 +587,7 @@ impl DebugProbe for JLink {
         Ok(())
     }
 
-    fn get_interface_jtag(
+    fn get_riscv_interface(
```

Comment:
You change the order of descriptors here (get_riscv_interface vs get_interface_riscv). I am okay with this, I guess I am completely indifferent about which one is better, but I think then we should at least change the others too for consistency and now might be a good time, don't you think?

Reply:
I changed the order to make it more consistent with `get_arm_interface`, which other methods do you mean?

Reply:
`get_interface_swo` for example. But I guess in fact just this one us the wrong way around. Maybe we should change this :)

Reply:
I've renamed it as well.